parquet-tools/src/main/java/parquet/tools/command/SizeCommand.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2013 ARRIS, Inc.
```

Comment:
Could you use the Apache copyright header? You can copy it from files in other parquet modules.